### DISCUSSION TITLE
Speed up initialization

### DISCUSSION THREAD

**Discussion by randyesperben** [MiczFlor/RPi-Jukebox-RFID#2392] [Q&A]
Hi!

I'm finishing the box using v3, and works very well. Just wondering if there is any way to speed up the initialization of system. I'm using RB4, and now it takes around 60 secs to start. Any idea?

Many thanks!

**Comment by randyesperben** [comment]
This is my boot services time chart. What are the essential jukebox services?

![carga_inicial](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/23616778/a718cd94-8fff-4459-96ce-39ef83225fae)

**Reply by s-martin** [reply]
You could check, if using a static IP improves your boot time as the dhcpd service takes 14 seconds.

**Reply by randyesperben** [reply]
Yes, when I'm connected via Ethernet the speed is around 9secs, but when disconnect the cable and connect via WIFI it takes around 20secs. Is this configuration ok?

<img width="309" alt="image" src="https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/23616778/30edb229-ad29-4efe-a593-797841213843">

**Reply by randyesperben** [reply]
I rectify, connected via cable the speed seems to be correct (see image), but doing the real test, since I turn on the raspberry, until it reads the nfc card and plays music, it is not until the second 60 approximately. what can it be?

<img width="488" alt="image" src="https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/23616778/451aa746-f086-4724-a772-ae74de28cfc6">

**Reply by AlvinSchiller** [reply]
Do you have the autohotspot feature installed?

**Reply by randyesperben** [reply]
Nope.
This is my systemd chart now (using Wifi, one nfc reader, just one song/card configured, but the reality it takes much more time to start playing:

![boot](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/23616778/50a02860-0ed6-449a-85bd-131c59687f4a)

**Reply by pabera** [reply]
Indeed, the start up is a problem. And most likely the biggest time might be consumed by the jukebox services. We always wanted to improve this but never had the time.

In your chart, I see the Phoniebox services listed. How come?

**Reply by randyesperben** [reply]
So, that means the jukebox being ready to play usually takes 60 seconds on average?

Regarding my chart, which are the jukebox services? I don't have anything specially configured, it's a clean installation.

**Reply by s-martin** [reply]
> In your chart, I see the Phoniebox services listed. How come?

I guess you meant you **don’t** see them listed?
